arbos/l2pricing/model.go

```diff
@@ -149,30 +151,10 @@ func applyGasDelta(op BacklogOperation, backlog uint64, delta uint64) uint64 {
 	}
 }
 
-// TODO(NIT-4152): eliminate manual gas calculation
 // BacklogUpdateCost returns the gas cost for updating the backlog in the active pricing model.
 func (ps *L2PricingState) BacklogUpdateCost() uint64 {
```

Comment:
It is worth adding a comment here stating this function won't be called after ArbOS60.